src/nextjournal/clerk/sci_env.clj

```diff
@@ -0,0 +1,8 @@
+(ns nextjournal.clerk.sci-env)
+
+(defmacro sci-copy-nss [& nss]
+  (into {} (for [[_ ns] nss]
+             [(list 'quote ns) `(sci.core/copy-ns ~ns (sci.core/create-ns '~ns))])))
```

Comment:
@borkdude any interest in getting such a convenience macro or something similar into sci?

Reply:
or would it be possible to write this as a function if `sci.core/copy-ns` also accepted a quoted symbol as the first arg?

Reply:
I don't intend to add this macro today since it hides the ability to pass in the result of `sci/create-ns` which can carry metadata etc.

You have to do this using a macro since `sci.core/copy-ns` is also a macro.

Reply:
or if not a quoted symbol, maybe a namespace:
```clojure
(into {}
      (map (fn [ns-sym] [ns-sym (sci.core/copy-ns (find-ns ns-sym) (sci.core/create-ns ns-sym))]))
      '[nextjournal.clerk.parser nextjournal.clerk.render])
```